torch/csrc/jit/runtime/operator.h

```diff
@@ -29,7 +29,7 @@ struct Node;
 using ::c10::FunctionSchema;
 using ::c10::Symbol;
 
-using OperationCreator = Operation (*)(const Node*);
+using OperationCreator = std::function<Operation(const Node*)>;
```

Comment:
I made this change in order to capture the `backend` struct in the lambdas registered as ops for `compile` and `execute`. I'd prefer not to make sure a wide-reaching change though, so please let me know if there are better alternatives.